LEAF_Request_Portal/templates/reports/LEAF_Sitemap_Search.tpl

```diff
@@ -981,6 +981,24 @@ function buildURLComponents(baseURL, update){
     window.history.pushState('', '', url);
 }
 
+function getPortalURL(url = '') {
```

Comment:
I think it's better not to for now, since the Vue app is its own file and importing another JS file to use the method there seems more complicated.  If this turns out to be needed in some additional places, then we can figure out the best place to put it.  It might also be better to circle back to this after the JS file moves are finished anyway.